pkg/utils/requester_test.go

```diff
@@ -190,12 +190,41 @@ func TestPostObj(t *testing.T) {
 	assert.NotNil(t, err)
 }
 
+type mockLogger struct {
+	Errors []error
+}
+
+func (m *mockLogger) Debug(message string)   {}
+func (m *mockLogger) Info(message string)    {}
+func (m *mockLogger) Warning(message string) {}
+func (m *mockLogger) Error(message string, err interface{}) {
+	if err, ok := err.(error); ok {
+		m.Errors = append(m.Errors, err)
+	}
+}
+
 func TestGetBad(t *testing.T) {
+	// Using a mockLogger to ensure we're logging the expected error message
+	mLogger := &mockLogger{}
+	httpreq := NewHTTPRequester(mLogger)
 
-	httpreq := NewHTTPRequester(logging.GetLogger("", ""))
-	_, _, _, err := httpreq.Get("blah12345/good")
-	_, ok := err.(*url.Error)
+	badURL := "htp:/ww.bad-url.fake/blah12345"
```

Comment:
Is this better served as "http://ww.bad-url.fake/blah12345"

Reply:
agreed, updated